Website/hugo/data/questions-policy-compliance.yaml

```diff
@@ -0,0 +1,133 @@
+    answers:
+        - A:
+            answer: 'No'
+            weight: 0
+        - B:
+            answer: Yes, we do it when requested
+            weight: 0.2
+        - C:
+            answer: Yes, we do it every few years
+            weight: 0.5
+        - D:
+            answer: Yes, we do it at least annually
+            weight: 1
+    quality_criteria:
+        - You have procedures (automated, if possible) in place, to regularly generate compliance reports
+        - You have ensured compliance reports are delivered to all relevant stakeholders
+        - Stakeholders use the reported compliance status information to identify areas for improvement
+
+- Do you have a complete picture of your external compliance obligations?:
+    security_practice: Policy & Compliance
+    stream: B
+    stream_title: Compliance Management
+    maturity: 1
+    answers:
+        - A:
+            answer: 'No'
+            weight: 0
+        - B:
+            answer: Yes, for some of the applications
+            weight: 0.2
+        - C:
+            answer: Yes, for at least half of the applications
+            weight: 0.5
+        - D:
+            answer: Yes, for most of the applications
+            weight: 1
+    quality_criteria:
+        - You have identified all sources of external compliance obligations
+        - You have captured and reconciled compliance obligations from all sources
+
+- Do you have a standard set of security requirements, and verification procedures, addressing the organization's external compliance obligations?:
+    security_practice: Policy & Compliance
+    stream: B
+    stream_title: Compliance Management
+    maturity: 2
+    answers:
+        - A:
+            answer: 'No'
+            weight: 0
+        - B:
+            answer: Yes, for some of the applications
+            weight: 0.2
+        - C:
+            answer: Yes, for at least half of the applications
+            weight: 0.5
+        - D:
+            answer: Yes, for most of the applications
+            weight: 1
+    quality_criteria:
+        - You have mapped each external compliance obligation to a well-defined set of application requirements
+        - You have defined verification procedures, including automated tests (when possible), to verify compliance with compliance-related requirements
+
+- Do you regularly report on adherence to external compliance obligations, and use that information to guide efforts to close compliance gaps?:
+    security_practice: Policy & Compliance
+    stream: B
+    stream_title: Compliance Management
+    maturity: 3
+    answers:
+        - A:
+            answer: 'No'
+            weight: 0
+        - B:
+            answer: Yes, for some of the applications
+            weight: 0.2
+        - C:
+            answer: Yes, for at least half of the applications
+            weight: 0.5
+        - D:
+            answer: Yes, for most of the applications
+            weight: 1
+   quality_criteria:
```

Comment:
This line needs an extra space at the beginning to match the indentation used elsewhere. YAML is horribly picky it turns out :/

Reply:
Line 130 has 3 spaces indentation - it needs 4.

Reply:
That's really interesting, given the fact that I didn't change the indentation anywhere in the file. I started with a copy of the YAML content in a different question set, and changed content only to the right of colons...

Reply:
Space added, and updated file committed.